user-guide/Advanced_Functionality/Databases/Amazon_Keyspaces_Service.md

```diff
@@ -47,6 +47,38 @@ For Amazon Keyspaces, the Starfield certificate must be present on the local Win
 
    ![Add Certificate](~/user-guide/images/aks_add_certificate.png)
 
+## Generating credentials for Amazon Keyspaces
+
+In the Identity and Access Management (IAM) Service, you can generate new credentials that are used to access the Amazon Keyspaces service. Those credentials are then linked to the account that created them.
+
+In order to do this, the right permissions are required.
```

Comment:
Which permissions? This seems a bit too vague to be user-friendly. Or does this refer to the permissions you need to check in step 3 below? If it's the latter, I would leave out this sentence here, because it will likely put the reader on the wrong foot, making them think they need specific permissions before they can start looking at the procedure below.

Reply:
Removed this line.